### ISSUE TITLE
`appendOnly: true` doesn't seem to work

### ISSUE THREAD

**Issue by jrfnl** [srvaroa/labeler#104]
I'm experimenting with this action at the moment and noticed the following:

### Given a `labeler.yml` file containing the following:
```yml
version: 1
appendOnly: true
labels:
- label: "Status: triage"
  draft: False
  author-can-merge: False
- label: "Type: bug"
  draft: False
  body: ".* bug .*"
- label: "Type: documentation"
  draft: False
  files:
  - "/Docs/[A-Za-z0-9-]*/.*Standard.xml$"
```

### This happens:

When a new PR is opened and the `Type: bug` label is manually added, but the body of the PR does not contain the term "bug" (though the title does), the `Type: bug` label is removed from the PR by the workflow.

Same thing with the `Type: documentation` label. If this was manually added on creating the PR, the workflow removes it when the `files` regex doesn't match any files.

### While I expected:

... the `appendOnly: true` setting to prevent this from happening.

Please let me know if you need more information and thank you for creating this action runner!

**Comment by srvaroa** [comment]
Thanks for reporting it @jrfnl , I will look into this

**Comment by jrfnl** [comment]
@srvaroa Much appreciated!

**Comment by srvaroa** [comment]
@jrfnl this should be fixed as of [v1.7.2](https://github.com/srvaroa/labeler/releases/tag/v1.7.2) (or on the floating v1 if you're already on it).

**Comment by jrfnl** [comment]
Thank you @srvaroa ! I'm currently working on another workflow, once that's done, I'll switch this one to `v1`.
